src/components/FullTags/FullTagPage.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="canvas-size overflow">
+    <div v-for="(image, index) in imageUrls.length" :key="index">
+      <div class="image-size">
+        <img :src="imageUrls[index].url" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import dataFetch from "../../api/dataFetching";
+
+export default {
+  props: {
+    tags: Array,
+  },
+  data() {
+    return {
+      imageUrls: [],
+    };
+  },
+  created() {
+    this.tags.forEach((id) => {
+      dataFetch.getImageById(id).then((data) => {
+        if (data.length > 0) {
+          this.imageUrls.push({ id: id, url: data[0].imagePaths.square });
+        }
+      });
+    });
+  },
```

Comment:
I would assume that calling the API to get images for a specific tag would still be more optimal that calling the API for each image if with the tag. 
If we want to save server requests, the tag object should contain the ressource id (or url) in addition to the image id and then we can construct the image link directly without calling the API

Reply:
I implement the first version like that due to a fear regarding the performance. After a test, that works well with the proposed approach and the change is done.